package.json

```diff
@@ -28,6 +28,7 @@
     "link": "./scripts/npm-link.sh",
     "lint": "./node_modules/.bin/eslint \"**/*.{js,jsx}\"",
     "lint-fix": "./node_modules/.bin/eslint --fix \"**/*.{js,jsx}\"",
+    "lint:fix": "./node_modules/.bin/eslint --fix \"**/*.{js,jsx}\"",
```

Comment:
While I agree with this new name format, it doesn't match the format of the rest of the script names.  Put this back to `lint-fix`, or change the names of _all_ the scripts to have `:` instead of `-`s.